README.md

```diff
@@ -120,7 +120,30 @@ The proxy addresses, which you'll need to interact with your upgradeable contrac
 
 Open the `package.zos.<network>.json` and use the addresses found there to interact with your deployed contracts. Congratulations! The first version of your upgradeable smart contract app is deployed in the blockchain!
 
+## Using a standard library
+
+In addition to creating proxies for your own contracts, you can also re-use already deployed contracts from a zeppelin_os standard library. To do so, run the following command, with the name of the npm package of the stdlib you want to use. For example:
+
+```bash
+zos set-stdlib openzeppelin-zos --network [NETWORK]
+```
+
+The next `sync` operation will connect your application with the chosen standard library on the target network. From there on, you can create proxies for any contract provided by the stdlib:
+
+```bash
+zos create-proxy MintableToken --network [NETWORK]
+```
+
+However, in ganache on development, the standard library is not already deployed, since you are running from an empty blockchain. To work around this, you can run the following command:
```

Comment:
However, if you're using development nodes (such as `testrpc` or `ganache`), the standard ...